documentation/modules/file-system-overhead.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: CONCEPT
+[id="file-system-overhead_{context}"]
+= File system overhead
+ifdef::rhv[]
+When you run a cold migration from {rhv-full} to the {ocp} cluster that {project-full} is deployed on, the migration allocates persistent volumes without Customized Data Importer (CDI). In these cases, you might need to adjust the file system overhead.
+endif::[]
+
+
+ifdef::ostack[]
+When you migrate from {osp} to the {ocp} cluster that {project-full} is deployed on, the migration allocates persistent volumes without Customized Data Importer (CDI). In these cases, you might need to adjust the file system overhead.
+endif::[]
+
+The configured file system overhead has a default value of 10%. If that is too low, the disk transfer fails due to lack of space. In this case, increase the file system overhead.
+
+If you need to reduce storage consumption, decrease the file system overhead.
+
```

Comment:
I changed lines 17-19 just now.

Reply:
Sentences are all good. :-)  If the file system fails, they increase the overhead, as mentioned.  The next sentence says that "if you need to reduce storage consumption," decrease the overhead.  But, if I'm reading this right, if I decrease the overhead to reduce storage consumption, I risk the disk transfer failing.  Do we tell them anywhere the scenario where they would decrease the overhead but not have the disk transfer fail?

Reply:
@mnecas @fabiand Please advise.

Reply:
Generally would not recommend reducing that overhead as it could cause failed migrations, so I think we can remove that sentence. But I would highlight that this applies **only** for the volumes on the PVC with volumeMode Filesystem.

Reply:
@mnecas Changed.